platform/i18n/src/locales/pt-BR/UserPreferencesModal.json

```diff
@@ -1,6 +1,6 @@
 {
   "Cancel": "Cancelar",
-  "Reset to Defaults": "Restaurar Default",
+  "Reset to Defaults": "$t(Common:Reset) para Padrão",
```

Comment:
I'm wary of mixing common words and phrases. Is there any risk of "common:reset" not making sense in this context? Is there a benefit to reusing the common definition here?

Curious if Locize spells out best practices here.

Reply:
I've checked some posts online, to get an opinion about it, couldn't find any best practice related, but I would agree with you Danny, I am also wary about mixing common definitions this way. 

Its being used a lot in the i18n by the way.